Doc/library/struct.rst

```diff
@@ -367,6 +360,12 @@ Notes:
    For the ``'E'`` and ``'C'`` format characters, the packed representation uses
    the IEEE 754 binary32 and binary64 format for components of the complex
    number, regardless of the floating-point format used by the platform.
+   Note that support for complex types is available unconditionally,
+   regardless on support IEC 60559 compatible complex arithmetic (Annex G of
+   the C11 standard) by the compiler.  Each complex type is interpreted as an
+   array type containing exactly two elements of the corresponding real type
```

Comment:
I don't really like "corresponding real type" followed by "real and imaginary parts". Maybe something  like:

```rst
Each complex type is interpreted as an array of two elements corresponding to the real and imaginary parts respectively and of corresponding C type (float or double).
```